src/lib/accessibility/Behaviors/Toolbar/ToolbarBehavior.ts

```diff
@@ -0,0 +1,13 @@
+import { Accessibility } from '../../interfaces'
+
+const ToolbarBehavior: Accessibility = (props: any) => ({
+  attributes: {
+    root: {
+      role: 'toolbar',
+      'aria-label': props['aria-label'],
```

Comment:
I think this is not needed. It will come out of the box through the rest/unhandledProps.

Reply:
this one too